testing/bootstrap.sh

```diff
@@ -1,11 +1,28 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 #!/bin/bash
 #
 # This script is used to bootstrap the [prow
 # jobs](https://github.com/kubernetes/test-infra/tree/master/prow), which is
 # the Kubernetes open source test-runner, rather like Jenkins. In other words,
 # this pulls the Minigo repo at some point (i.e., at the a pull request or
 # commit sha) and then runs the Minigo tests.
-set -e
+
+set -o errexit
```

Comment:
why?

Reply:
For explicitness. Mostly copied from prow scripts, but I rather like it

Reply:
ok.